pkgs/by-name/db/dbvisualizer/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  copyDesktopItems,
+  fetchurl,
+  makeDesktopItem,
+  makeWrapper,
+  stdenv,
+  temurin-jre-bin-17,
+}:
+
+let
+  pname = "dbvisualizer";
+in
+stdenv.mkDerivation (finalAttrs: {
+  inherit pname;
+  version = "24.2.2";
+
+  src =
+    let
+      underscoreVersion = lib.concatStringsSep "_" (lib.splitString "." finalAttrs.version);
```

Comment:
You can do `builtins.replaceStrings [ "." ] [ "_" ] finalAttrs.version`

Reply:
Sure, that looks nicer!